batch/batch/worker/worker.py

```diff
@@ -1488,6 +1492,8 @@ def __init__(
 
         self.timings: Timings = Timings()
 
+        self.env['HAIL_REGION'] = REGION
```

Comment:
I think this should be 
```suggestion
        self.env.append({'name': 'HAIL_REGION', 'value': REGION})
```